node/blockchain/utils.py

```diff
@@ -0,0 +1,19 @@
+from contextlib import ContextDecorator
+
+from node.core.exceptions import BlockchainLockedError
+
+
+class TableLocker(ContextDecorator):
+
+    def __init__(self, model):
+        self.db_table_name = model._meta.db_table
+        self.table_lock = None
+
+    def __enter__(self):
+        from node.blockchain.models import TableLock
+        self.table_lock, created = TableLock.objects.get_or_create(name=self.db_table_name)
```

Comment:
is_created

Reply:
I think "created" is good enough too. Djano itself use "created" in code.